Makefile

```diff
@@ -1,23 +1,42 @@
+
+.DEFAULT_GOAL		:=help
+SHELL				:=/bin/bash
+
 .PHONY: help
 help: ## Show this help
 	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+##@ Set environment and corresponding configuration
+.PHONY: dev
 dev:
 	$(eval DEPLOY_ENV=dev)
 	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+	$(eval AZURE_BACKUP_STORAGE_ACCOUNT_NAME=s165d01dbbackup)
+	$(eval AZURE_BACKUP_STORAGE_CONTAINER_NAME=find-a-lost-trn)
 
+.PHONY: test
 test:
 	$(eval DEPLOY_ENV=test)
 	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
 
+.PHONY: preprod
 preprod:
 	$(eval DEPLOY_ENV=preprod)
 	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
 
+.PHONY: production
 production:
 	$(eval DEPLOY_ENV=production)
 	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-production)
+	$(eval AZURE_BACKUP_STORAGE_ACCOUNT_NAME=s165p01dbbackup)
```

Comment:
You could use `grep` to get these values from the `<env>.backend.tfvars` files similar to the way that `jq` is used elsewhere in this file to get values from `<env>.tfvars.json` files.  Something like:

`cat terraform/workspace_variables/dev.backend.tfvars | grep -Po "storage_account_name = \"\K\w+"`

Should work and would avoid duplication